recipes-bsp/u-boot/u-boot-socfpga_v2016.11.bb

```diff
@@ -2,7 +2,7 @@ require u-boot-socfpga-common.inc
 require u-boot-socfpga-env.inc
 require ${COREBASE}/meta/recipes-bsp/u-boot/u-boot.inc
 
-FILESEXTRAPATHS =. "${THISDIR}/files/v2016.11:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/files/v2016.11:"
```

Comment:
im not sure i understand the question.  actually, i didnt mean to change that.  what is the difference between .= and _append?